client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +89,208 @@ function HelpModal({
+    setDescription('');
+    setReadSearchCheckbox(false);
+    setSimilarQuestionsCheckbox(false);
+  };
+
+  const handleCheckboxChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    set: React.Dispatch<React.SetStateAction<boolean>>
+  ) => {
+    set(event.target.checked);
+  };
+
   if (isOpen) {
     callGA({ event: 'pageview', pagePath: '/help-modal' });
   }
   return (
-    <Modal dialogClassName='help-modal' onHide={closeHelpModal} show={isOpen}>
+    <Modal
+      dialogClassName='help-modal'
+      onHide={() => {
+        closeHelpModal();
+        setShowHelpForm(false);
+        resetFormValues();
+      }}
+      show={isOpen}
+    >
       <Modal.Header className='help-modal-header fcc-modal' closeButton={true}>
         <Modal.Title className='text-center'>
           {t('buttons.ask-for-help')}
         </Modal.Title>
       </Modal.Header>
       <Modal.Body className='help-modal-body text-center'>
-        <h3 className='help-modal-heading'>
-          <Trans i18nKey='learn.tried-rsa'>
-            <a
-              href={RSA}
-              rel='noopener noreferrer'
-              target='_blank'
-              title={t('learn.rsa')}
+        {showHelpForm ? (
+          <form
+            onSubmit={event => {
+              event.preventDefault();
+              if (!canSubmitForm()) return;
+              setShowHelpForm(false);
+              resetFormValues();
+              createQuestion();
+              closeHelpModal();
+            }}
+            ref={formRef}
+          >
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column'
+              }}
+            >
+              <fieldset>
+                <legend style={{ fontSize: 'large' }}>
+                  {t('learn.must-confirm-statements')}
+                </legend>
+                <CheckboxHelpModal
+                  name='read-search-ask-checkbox'
+                  i18nkey={t('learn.read-search-ask-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setReadSearchCheckbox)
+                  }
+                  value={readSearchCheckbox}
+                  href={RSA}
+                  t={t}
+                />
+                <Spacer size='small' />
+                <CheckboxHelpModal
+                  name='similar-questions-checkbox'
+                  i18nkey={t('learn.similar-questions-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setSimilarQuestionsCheckbox)
+                  }
+                  value={similarQuestionsCheckbox}
+                  href={generateSearchLink(challengeTitle, challengeBlock)}
+                  t={t}
+                />
+              </fieldset>
+              <label htmlFor='help-modal-form-description'>
+                {t('forum-help.whats-happening')}
+                <span className='sr-only'>50 character minimum</span>
```

Comment:
```suggestion
                <span className='sr-only'>{t('learn.min-50-max-500')}</span>
```

This needs to be translated.